typescript/src/metals.ts

```diff
@@ -74,6 +78,16 @@ export class Metal {
   getProperty(key: keyof Metal) {
     return this[key];
   }
+
+  disableMetal() {
```

Comment:
I wonder if it's worth either renaming this method or adding a comment explaining what it's doing? (Perhaps check the wording Andrew uses in his emails.)